lib/iris/pandas.py

```diff
@@ -124,65 +124,7 @@ def _as_pandas_coord(coord):
     return index
 
 
-def _assert_shared(np_obj, pandas_obj):
-    """Ensure the pandas object shares memory."""
-    values = pandas_obj.values
-
-    def _get_base(array):
-        # Chase the stack of NumPy `base` references back to the original array
-        while array.base is not None:
-            array = array.base
-        return array
-
-    base = _get_base(values)
-    np_base = _get_base(np_obj)
-    if base is not np_base:
-        msg = "Pandas {} does not share memory".format(
-            type(pandas_obj).__name__
-        )
-        raise AssertionError(msg)
-
-
-def as_series(cube, copy=True):
-    """
-    Convert a 1D cube to a Pandas Series.
-
-    Args:
-
-        * cube - The cube to convert to a Pandas Series.
-
-    Kwargs:
-
-        * copy - Whether to make a copy of the data.
-                 Defaults to True. Must be True for masked data.
-
-    .. note::
-
-        This function will copy your data by default.
-        If you have a large array that cannot be copied,
-        make sure it is not masked and use copy=False.
-
-    """
-    data = cube.data
-    if ma.isMaskedArray(data):
-        if not copy:
-            raise ValueError("Masked arrays must always be copied.")
-        data = data.astype("f").filled(np.nan)
-    elif copy:
-        data = data.copy()
-
-    index = None
-    if cube.dim_coords:
-        index = _as_pandas_coord(cube.dim_coords[0])
-
-    series = pandas.Series(data, index)
-    if not copy:
-        _assert_shared(data, series)
-
-    return series
-
-
-def as_data_frame(cube, copy=True):
+def as_data_frame(cube, dropna=True, asmultiindex=False):
```

Comment:
```suggestion
def as_data_frame(cube, dropna=False, asmultiindex=False):
```

Might be better, depending on what you think of my other comments in this review. Although I'm still not certain.